shared/templates/mount_option_home/oval.template

```diff
@@ -0,0 +1,75 @@
+{{% set non_allowed_partitions = ["/lib",
+                                  "/opt",
+                                  "/usr",
+                                  "/bin",
+                                  "/sbin",
+                                  "/boot",
+                                  "/dev",
+                                  "/proc"] %}}
+<def-group>
+    <definition class="compliance" id="{{{ _RULE_ID }}}" version="1">
+        {{{ oval_metadata("Home mount points should be mounted with mount option " + MOUNTOPTION + ".") }}}
+        <criteria>
```

Comment:
Please, include explicit `operator` and a `comment`, so it helps reading logs and debugging.